### PR TITLE
Add Snapshot index and Pruning index to Prometheus

### PR DESCRIPTION
**Pull request by c-mnd**
# Description

Adds Snapshot index as `iota_info_snapshot_index`to the Prometheus plugin.
Adds Pruning index as `iota_info_pruning_index` to the Prometheus plugin.

## Type of change

- [x] Enhancement (non-breaking change which adds functionality)

# How Has This Been Tested?

Both metrics show up in the Prometheus expression browser correctly.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have tested my code extensively
- [x] I have selected the `develop` branch as the target branch

### CODE REVIEW

File: plugins/prometheus/info.go [hunk 1]
```diff
@@ -105,6 +117,20 @@ func collectInfo() {
 		cachedSolidMs.Release()
 	}
 
+	snapshotInfo := tangle.GetSnapshotInfo()
+
+	// Snapshot index
+	if snapshotInfo != nil {
+		si := tangle.GetSnapshotInfo().SnapshotIndex
```
Comment (muXxer) [inline plugins/prometheus/info.go]: You can set both values in the same if condition. No need to check again.

And you could set it directly, for example `infoSnapshotIndex.Set(float64(tangle.GetSnapshotInfo().SnapshotIndex))`

**Review by muXxer — CHANGES_REQUESTED** [review]

File: plugins/prometheus/info.go (hunk 1 above)
Comment (c-mnd) [reply plugins/prometheus/info.go]: The style should follow the rest of this plugin, that's why I did it seperately.

Comment (muXxer) [reply plugins/prometheus/info.go]: There is no line in the code where we check the same condition twice. Please change it ;)

**Comment by c-mnd** [comment]
But yeah, will do so!

File: plugins/prometheus/info.go (hunk 1 above)
Comment (c-mnd) [reply plugins/prometheus/info.go]: But yeah, will do so!

Comment (c-mnd) [reply plugins/prometheus/info.go]: But I'd rather use previously defined `snapshotInfo` var
`infoSnapshotIndex.Set(float64(snapshotInfo.SnapshotIndex))`
than calling `GetSnapshotInfo` again by
`infoSnapshotIndex.Set(float64(tangle.GetSnapshotInfo().SnapshotIndex))`
? right?

Comment (muXxer) [reply plugins/prometheus/info.go]: Ah, sure! :)
Thats better!

**Review by muXxer — APPROVED** [review]

File: plugins/prometheus/info.go (hunk 1 above)
Comment (c-mnd) [reply plugins/prometheus/info.go]: Think I'm done with requested changes.
Now is it me or you @muXxer to click "Resolve conversation"? Not sure.

**Pull request merged**